script/openqa-clone-job

```diff
@@ -66,6 +66,10 @@ do not clone parent jobs of type chained. This makes the job use the downloaded
 
 do not try any download. You need to ensure all required assets are provided yourself.
 
+=item B<--clone-children>
+
+clone child jobs as well. This is useful if you want to clone a full multi-machine clusters.
```

Comment:
I would appreciate a bit more detail here. Because to my understanding - as you already discussed with mkittler in review comments as well - cloning "full multi-machine clusters" is already available by cloning based on a child. You could explain a bit better that this is related to cloning based on a *parent*, right?

Reply:
> cloning "full multi-machine clusters" is already available by cloning based on a child

Maybe I misunderstood something or I not explained well: but currently we can't clone a "full multi-machine cluster", only half of it (which is not very useful and don't work ;-))

> You could explain a bit better that this is related to cloning based on a _parent_, right?

Yes the parent ID is passed and cloned, as well as his child. I will add a little more details here.

Reply:
> cloning "full multi-machine clusters" is already available by cloning based on a child

No - that's not true. If one clones a child that covers the parallel parent but not parallel siblings.

It is obvious that with this option one needs to clone the parallel parent instead (otherwise it would be `--clone-siblings`).

Reply:
I added a comment in `usage`.